index.js

```diff
@@ -455,7 +455,7 @@ class Hyperbee {
     this.core.off('append', this._onappendBound)
 
     for (const watcher of this._watchers) {
-      watcher.destroy()
+      await watcher.destroy()
```

Comment:
Can be `await Promise.all(this._watchers.map(w => w.destroy()))`

Reply:
Actually shouldnt matter here as the actual async time is spent closing the snaps so parallel is the same as serial time minus the ticks, but given the current simplicity i prefer that